cmake/toolchains/GCC_ARM.cmake

```diff
@@ -31,8 +31,8 @@ list(APPEND common_options
     "-funsigned-char"
     "-MMD"
     "-fomit-frame-pointer"
-    "-Os"
     "-g"
+    "-g3"
```

Comment:
We don't need both `-g` and `-g3`, or do we?

Reply:
Removed